components/scream/src/physics/p3/p3_constants.hpp

```diff
@@ -0,0 +1,63 @@
+#ifndef P3_CONSTANTS_HPP
+#define P3_CONSTANTS_HPP
+
+#include "share/scream_types.hpp"
+
+#include <vector>
+
+namespace scream {
+namespace p3 {
+
+/*
+ * Mathematical constants used by p3.
+ */
+
+template <typename Scalar>
+struct Constants
+{
+  static constexpr Scalar INV_RHOW = 1.e-3;
```

Comment:
Not sure, it was like this in fortran p3.

Reply:
I agree. What I meant to say is that I don't know what the intent is behind this variable since I'm not an expert in the original p3 impl. Here's where it was being used:

```
prt_liq(i) = prt_liq(i) + prt_accum*INV_RHOW*odt
```

Reply:
*Brought over from SLACK* 

`prt_liq` is used by the mass checker to figure out how much mass has left the system via the surface precip
`prt_accum` is the sedimentation at the surface

Reply:
per @bartgol comment: In the implementation that will eventually make it into SCREAM we have fixed this inconsistentcy.  In short, yes inv_rhow should not be defined this way, instead it should be 1.0/rhow which we have done

Reply:
@AaronDonahue is there a reason for INV_RHOW to not equal 1.0/RHOW?

Reply:
Not really.  I think the new implementation of P3 that Hugh and his team just recently released fixes this to be 1.0/RHOW.

Reply:
``` cp     = 1005.
 inv_cp = 1./cp
 g      = 9.816
 rd     = 287.15
 rv     = 461.51
 ep_2   = 0.622
 rhosur = 100000./(rd*273.15)
 rhosui = 60000./(rd*253.15)
 ar     = 841.99667
 br     = 0.8
 f1r    = 0.78
 f2r    = 0.32
 ecr    = 1.
 rhow   = 1000.
 cpw    = 4218.
 inv_rhow = 1./rhow  !inverse of (max.) density of liquid water
 mu_r_constant = 1.  !fixed shape parameter for mu_r```

Reply:
@jgfouca then let's break BFB w.r.t. master with this change, too: define INV_RHOW as 1.0/RHOW.

Reply:
Will it make much difference to your effort when we bring in the P3 branch which will have a set of changes to this set of constants?  Would it be helpful for me to share the constants we plan on using in the final version as they are defined now, or would switching over be just as easy when we get to the point that we are bringing in P3?

Reply:
for example, many of these constants will be changed to adopt the E3SM set of universal constants.

Reply:
@AaronDonahue, I think it's OK to leave the changes to the constants for later. This PR focuses on mostly low-level code. The issue of constants came up only b/c they affected the ICs for the current P3 baseline regression test.